aiosmtpd/tests/test_smtp.py

```diff
@@ -19,6 +19,15 @@ def factory(self):
         return Server(self.handler, decode_data=False)
 
 
+class ReceivingHandler:
+    box = None
+
+    def process_message(self, *args, **kws):
+        if not self.box:
+            self.box = []
+        self.box.append(args)
```

Comment:
I think it would be better to have an `__init__()` that sets `self.box = []` so the value will always be a list even if `process_message()` is never called.  Then you can also get rid of the class attribute.